crates/type-system/scripts/build-wasm.ts

```diff
@@ -0,0 +1,80 @@
+import fs from "fs-extra";
+import * as child_process from "node:child_process";
```

Comment:
We can use `execa` instead, which has a slightly more convenient API. No need to do this now, this is just an idea.

Reply:
Have captured as a @todo